### PR TITLE
[zh] Replace English CNCF code of conduct with Chinese one

### PR DESCRIPTION
**Pull request by rayw000**
Current CNCF code of conduct under `/content/zh` directory is in English. Change it with Chinese edition.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks for suggesting this change, @rayw000!

File: content/zh/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -1,46 +1,30 @@
...
+## 云原生计算基金会（CNCF）社区行为准则 1.0 版本
 
-### Contributor Code of Conduct
+### 贡献者行为准则
 
-As contributors and maintainers of this project, and in the interest of fostering
-an open and welcoming community, we pledge to respect all people who contribute
-through reporting issues, posting feature requests, updating documentation,
-submitting pull requests or patches, and other activities.
+作为这个项目的贡献者和维护者，为了建立一个开放和受欢迎的社区，我们保证尊重所有通过报告问题、发布功能请求、更新文档、提交拉取请求或补丁以及其他活动做出贡献的人员。
```
Comment (helight) [inline content/zh/community/static/cncf-code-of-conduct.md:7]: pull request 是不是不应该翻译呀
```suggestion
作为这个项目的贡献者和维护者，为了建立一个开放和受欢迎的社区，我们保证尊重所有通过报告问题、发布功能请求、更新文档、提交 pull request（PR）或补丁以及其它活动做出贡献的人员。
```

File: content/zh/community/static/cncf-code-of-conduct.md [hunk 2]
```diff
@@ -1,46 +1,30 @@
...
-* Public or private harassment
-* Publishing other's private information, such as physical or electronic addresses,
- without explicit permission
-* Other unethical or unprofessional conduct.
+-	使用性语言或图像
+-	人身攻击
+-	挑衅、侮辱或贬低性评论
+-	公开或私下骚扰
+-	未经明确许可，发布他人的私人信息，比如地址或电子邮箱
+-	其他不道德或不专业的行为
```
Comment (helight) [inline content/zh/community/static/cncf-code-of-conduct.md:18]:
```suggestion
-	其它不道德或不专业的行为
```

File: content/zh/community/static/cncf-code-of-conduct.md [hunk 3]
```diff
@@ -1,46 +1,30 @@
...
 
-Project maintainers have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are not
-aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers
-commit themselves to fairly and consistently applying these principles to every aspect
-of managing this project. Project maintainers who do not follow or enforce the Code of
-Conduct may be permanently removed from the project team.
+项目维护者有权利和责任删除、编辑或拒绝评论、提交、代码、维基编辑、问题和其他不符合本行为准则的贡献。通过采用本行为准则，项目维护者承诺将这些原则公平且一致地应用到这个项目管理的各个方面。不遵守或不执行行为准则的项目维护者可能被永久地从项目团队中移除。
```
Comment (helight) [inline content/zh/community/static/cncf-code-of-conduct.md:20]:
```suggestion
项目维护者有权利和责任删除、编辑或拒绝不符合本行为准则的评论、提交、编写代码、编辑维基、提问和其它贡献。通过采用本行为准则，项目维护者承诺将这些原则公平且一致地应用到这个项目管理的各个方面。不遵守或不执行行为准则的项目维护者可能被永久地从项目团队中移除。
```

File: content/zh/community/static/cncf-code-of-conduct.md [hunk 4]
```diff
@@ -1,46 +1,30 @@
...
 
-This code of conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community.
+当个人代表项目或其社区时，本行为准则适用于项目空间和公共空间。
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting
-the [Kubernetes Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct) <conduct@kubernetes.io>.
+如需举报侮辱、骚扰或其他不可接受的行为，您可发送邮件至 <conduct@kubernetes.io> 联系 [Kubernetes行为守则委员会](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct)。其他事务请联系CNCF项目维护专员，或发送邮件至 <mishi@linux.com> 联系我们的调解员Mishi Choudhary。
```
Comment (helight) [inline content/zh/community/static/cncf-code-of-conduct.md:24]:
```suggestion
如需举报侮辱、骚扰或其它不可接受的行为，可发送邮件至 <conduct@kubernetes.io> 联系 [Kubernetes 行为守则委员会](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct)。其他事务请联系 CNCF 项目维护专员，或发送邮件至 <mishi@linux.com> 联系我们的调解员 Mishi Choudhary。
```

**Comment by rayw000** [comment]
Hi @helight , Thanks for your review!
According to the first line of `/content/zh/community/static/cncf-code-of-conduct.md` suggests, this file should not be edited directly. It should be copied from https://github.com/cncf/foundation/blob/master/code-of-conduct.md . Perhaps you could post these modifications to https://github.com/cncf/foundation ? 😃

**Comment by helight** [comment]
> https://github.com/cncf/foundation

ok

**Pull request merged**